example/package.json

```diff
@@ -14,10 +14,10 @@
     "@react-navigation/native-stack": "~6.9.12",
     "@react-navigation/stack": "~6.3.16",
     "react": "18.2.0",
-    "react-native": "0.71.8",
+    "react-native": "0.72.3",
```

Comment:
[high]: Could you revert that file?

Reply:
When I tried to run the interpolate in the background Color it doesn't worked in 0.71.8 version. That's why I've upgraded the react native version. Maybe it's a specific issue from the current used version in the example...